File: armstrong.cpp
#include<iostream>
using namespace std;
int main()
{
    int n, num, rem, sum = 0;
    cout << "Enter a positive  integer: ";
    cin >> num;
    n = num;
    while(n != 0)
    {
        rem = n % 10;
        sum += rem * rem * rem;
        n /= 10;
    }
    if(sum == num)
        cout << num << " is an Armstrong number.";
    else
        cout << num << " is not an Armstrong number.";
    return 0;
}

File: armstrong.py
print ("Enter a positive integer: ")
num = int(input())
n = num
num1 = num
sum = 0
count = 0
while( n != 0):
    while(num1 > 0):
        count = count + 1
        num1 = num1 // 10
    rem = n % 10
    sum = sum + rem ** count
    n = n // 10
if (sum == num):
    print("The number is an Armstrong number")
else:
    print("The number is not an Armstrong number")